src/components/FilterOverlay.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+  <div id="dialog">
+    <v-app id="filterOverlayDialog">
```

Comment:
We only really need IDs to reference them elsewhere
```suggestion
  <div>
    <v-app>
```